send-to-bluetooth@asgharalitaj/files/send-to-bluetooth@asgharalitaj/send-to-bluetooth@asgharalitaj.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Check if file argument is provided
+if [ -z "$1" ]; then
+    echo "Usage: $0 <file>"
+    exit 1
+fi
+
+# File to send
+FILE="$1"
+
+# Choose Bluetooth device
+DEVICE=$(blueman-sendto --choose)
```

Comment:
This does not appear to be a supported option. Can you specify what version of blueman provides this? This will definitely be an issue for backward compatibility if it is a new option as there are quite a bit of users on older versions.

Reply:
i was pretty sure that it will work. Do you have any better suggestion?

Reply:
@AsgharAliTaj I guess the final command should be:

```bash
blueman-sendto $FILE
```

Remove the lines `DEVICE=$(blueman-sendto --choose)` and `blueman-sendto --send $FILE $DEVICE`, and add the line above instead.

Reply:
done

Reply:
> done

Have you tested this? You can run `./test-spice send-to-bluetooth@asgharalitaj` to test it locally. I would still suggest changing the name to `send-via-bluetooth` as it is much more descriptive and accurate.